index.html

```diff
@@ -16,33 +16,47 @@
 <body>
     <section class="header_headline_section">
         <header>
-            <nav>
+            <nav class="container">
                 <div class="header_logo"><a href="index.html" class="logo">My Logo</a></div>
-                <div class="header_icon"><img src="icons/Icon-Menu.png" alt="" srcset=""></div>
+                <div class="header_icon"><img src="icons/Icon-Menu.png" alt="menu icon" srcset=""></div>
+                <ul class="navbar">
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">Portfolio</a>
+                    </li>
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">About</a>
+                    </li>
+                    <li class="nav-list">
+                        <a href="#" class="nav-item">Contact</a>
+                    </li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] To further improve accessibility and user experience via your page navigation, adding internal links that redirect users to appropriate sections of the page (`portfolio`, `about`, and `contact`) will show your commitment to good UI/UX. You can implement this with the use of `id`'s.

_The navigation links are static at the moment, but could be made live if you include correct sectional id's in the `href` attribute._